fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/query/visitors/DdlVisitor.java

```diff
@@ -376,7 +376,7 @@ private RecordLayerView getViewMetadata(@Nonnull final RelationalParser.ViewDefi
         getDelegate().replaceSchemaTemplate(ddlCatalog);
 
         // 1. get the view name.
-        final var viewName = visitFullId(viewCtx.viewName).toString();
+        final var viewName = Identifier.toProtobufCompliant(visitFullId(viewCtx.viewName)).getName();
```

Comment:
Do view names actually need to be protobuf compliant? Shouldn't we only generate one of these if we're going to generate a protobuf ID out of the SQL ID (things like table names, structs, enums, fields, etc). I would have thought that that's not necessary for views